framework/db/ActiveRecord.php

```diff
@@ -268,7 +268,7 @@ public static function deleteAll($condition = '', $params = [])
      */
     public static function find()
     {
-        return Yii::createObject(ActiveQuery::className(), [get_called_class()]);
+        return Yii::createObject(ActiveQuery::class, [get_called_class()]);
```

Comment:
since `find` is called from the AR then `get_called_class()` and `static::class` should behave similary.
```php
User extends ActiveRecord
{
}

User::find()->modelClass // will return a query created using the `User` class.
```

https://3v4l.org/gZFEV here is a live example.